.travis.yml

```diff
@@ -0,0 +1,6 @@
+---
+language: ruby
+cache: bundler
+rvm:
+  - 2.7.1
+before_install: gem install bundler -v 2.1.4
```

Comment:
please remove this file

Reply:
It's because I replace that with .ruby-version?

Reply:
no. you ouse circleci for CI. Travis.yml not needed

Reply:
Got it. Thanks. I'll do it